cmd/chaos-builder/impl.go

```diff
@@ -97,6 +97,7 @@ func (in *{{.Type}}) GetChaos() *ChaosInstance {
 		Kind:      Kind{{.Type}},
 		StartTime: in.CreationTimestamp.Time,
 		Action:    "",
+		Status:    string(in.GetStatus().Experiment.Phase),
```

Comment:
Why we need use `in.GetStatus()` function here? Can we use `in.Status.xxxx` directly?